fms/modules/tp.py

```diff
@@ -73,8 +80,10 @@ def copy_rowwise(
             Parameter that has had TP applied
         tensor_value: torch.Tensor
             tensor that needs sharding
+        dim: int
+            Dimension on which to shard
```

Comment:
nit: we might want to explicitly say that 0 is a colwise shard and 1 means row-wise